src/styles/document-export-page.css

```diff
@@ -239,3 +240,10 @@
 .joubel-create-document.responsive.no-title .joubel-exportable-header-text {
   display: none;
 }
+
+
+@media only screen and (max-device-width: 1180px) {
```

Comment:
Can you use _just_ device detection rather than mix in media queries also? (see note above for why I think you ran into trouble)